Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node
+
+RUN mkdir /code
+WORKDIR /code
+
+ADD package.json /code
+RUN npm install
+
+ADD . /code
+
+EXPOSE 3000
+
+CMD ["node", "server.js"]
```

Comment:
`npm` does not pass signals it receives down to child processes so need to call `node` directly so hapi server will gracefully exit